lib/trip_dispatcher.rb

```diff
@@ -90,6 +91,97 @@ def load_trips
       trips
     end
 
+    def self.split_date_time(datetime)
+      #puts datetime.split("T")
+      puts Time.parse(datetime)
+    end
+
+    def available_drivers
+      available_drivers = []
+      @drivers.each do |driver|
```

Comment:
A `select` could do this more compactly.  There's no problem with the code here, but just a reminder that you can use Enumerable methods.